### PR TITLE
libiberty 11.1.0 (new formula)

### PR DESCRIPTION
**Pull request by curoky**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

The libiberty library is a collection of subroutines used by various GNU programs.

### CODE REVIEW

File: Formula/libiberty.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  def install
+    ENV.append_to_cflags "-fPIC"
+
+    args = %W[
+      --prefix=#{prefix}
+      --enable-install-libiberty
+    ]
+
+    mkdir "build" do
+      system "../libiberty/configure", *args
```
Comment (SMillerDev) [inline Formula/libiberty.rb:20]:
```suggestion
      system "../libiberty/configure", *std_configure_args, "--enable-install-libiberty"
```

File: Formula/libiberty.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  homepage "https://gcc.gnu.org/onlinedocs/libiberty"
+  url "https://ftp.gnu.org/gnu/gcc/gcc-11.2.0/gcc-11.2.0.tar.xz"
+  mirror "https://ftpmirror.gnu.org/gcc/gcc-11.2.0/gcc-11.2.0.tar.xz"
+  sha256 "d08edc536b54c372a1010ff6619dd274c0f1603aa49212ba20f7aa2cda36fa8b"
+  license "GPL-3.0-or-later" => { with: "GCC-exception-3.1" }
+
+  depends_on "gcc" => :build
+
+  def install
+    ENV.append_to_cflags "-fPIC"
```
Comment (SMillerDev) [inline Formula/libiberty.rb:12]: Why is this needed?

**Comment by fxcoudert** [comment]
Thanks for suggesting this, but we won't ship it in Homebrew core. You can maintain your [own tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap) if you have a use for it.

GNU libiberty is not really a standalone library, but rather, like gnulib, intended to be included at the source level into the projects that need it (mostly GCC and GNU developer tools). It is similar to gnulib:

> Gnulib does not make releases. It is intended to be used at the source level.

**Comment by curoky** [comment]
@fxcoudert ok, since it cannot be installed independently, can we install libiberty as part of GCC?

**Comment by fxcoudert** [comment]
@curoky the same arguments hold. It's meant to be included at the source level into the projects that need it, not used as a common library.
